utils/build/webpack.common.js

```diff
@@ -23,128 +18,157 @@ const IMAGES_DIR = path.resolve(__dirname, '../../client/Images');
+const withStylingModuleLoader = returnModuleRuleWithConfig(
+  {
     test: /(\.css|.scss)$/,
     sideEffects: true, // the css loader requires sideEffects true, else no built css is emitted
-}, [{
-    loader: 'css-loader',
-},
-{
-    loader: 'sass-loader',
-    options: {
+  },
+  [
+    {
+      loader: 'css-loader',
+    },
+    {
+      loader: 'sass-loader',
+      options: {
         sassOptions: {
-            includePaths: ['node_modules'],
+          includePaths: ['node_modules'],
         },
+      },
     },
-},]);
+  ]
+);
 
-const withJSModuleLoader = returnModuleRuleWithConfig({
+const withJSModuleLoader = returnModuleRuleWithConfig(
+  {
     test: /\.(jsx|js)?$/,
     exclude: /(node_modules)/,
-}, [{
-    loader: 'babel-loader',
-    options: {
+  },
+  [
+    {
+      loader: 'babel-loader',
+      options: {
         cacheDirectory: true,
         presets: babelPresets,
+      },
     },
-}, ]);
+  ]
+);
 
-const withFontModuleLoader = returnModuleRuleWithConfig({
+const withFontModuleLoader = returnModuleRuleWithConfig(
+  {
     test: /\.(woff(2)?|ttf|eot)$/,
-}, [{
-    loader: 'file-loader',
-    options: {
+  },
+  [
+    {
+      loader: 'file-loader',
+      options: {
         name: '[name].[ext]',
         publicPath: '/fonts/',
         outputPath: 'fonts/',
+      },
     },
-}, ]);
+  ]
+);
 
-const withImageModuleLoader = returnModuleRuleWithConfig({
+const withImageModuleLoader = returnModuleRuleWithConfig(
+  {
     test: /\.(jpg|gif|png|svg)$/,
-}, [{
-    loader: 'file-loader',
-    options: {
+  },
+  [
+    {
+      loader: 'file-loader',
+      options: {
         publicPath: '/images/',
         outputPath: 'images/',
+      },
     },
-}, ]);
+  ]
+);
 
 // exported helper function - returns common configuration, merged via object spread. Provided options take precedence
 const returnBasicConfigMergedWith = (customConfigurationForBuildMode = {}) => ({
-    entry: `${BOOTSTRAP_DIR}/index.js`,
-    output: {
-        path: BUILD_DIR,
-        publicPath: '',
-        filename: '[name].bundle.js',
-    },
-    module: {
-        // given the dynamic nature of rules and configuration, none a re provided by default. Use the exported `moduleLoaders` to return loaders with some default configs
-        rules: []
-    },
-        // given different build modes will require different plugins/plugin configuration, none are provided by default. Use the exported `plugins` for commonly used plugins with default configuration
-    plugins: [],
-    resolve: {
-        alias: webpackAliases
-    },
-    ...customConfigurationForBuildMode
+  entry: `${BOOTSTRAP_DIR}/index.js`,
+  output: {
+    path: BUILD_DIR,
+    publicPath: '',
+    filename: '[name].bundle.js',
+  },
+  module: {
+    // given the dynamic nature of rules and configuration, none a re provided by default. Use the exported `moduleLoaders` to return loaders with some default configs
+    rules: [],
```

Comment:
Do we need these extra commas at the end of all these json objects in webpack files?